From c52eb33f20a66acb648b725e2eee3ed977eae896 Mon Sep 17 00:00:00 2001
From: Antonio Rojas <arojas@archlinux.org>
Date: Tue, 3 Aug 2021 10:41:09 +0200
Subject: [PATCH] Only link to Imath when OpenEXR version is >= 3.0

Since OpenEXR 3.0, Imath is in a separate, independently released library. Since that's the only library that Calligra needs, there's no need to link to the full OpenEXR
---
 CMakeLists.txt | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index cdcc02f0236..91c72882d3a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -338,9 +338,9 @@ endif ()
 ## Check for OpenEXR
 ##
 
-macro_optional_find_package(OpenEXR 3.0 CONFIG QUIET)
-if(TARGET OpenEXR::OpenEXR)
-    set(OPENEXR_LIBRARIES OpenEXR::OpenEXR)
+macro_optional_find_package(Imath 3.0 CONFIG QUIET)
+if(TARGET Imath::Imath)
+    set(OPENEXR_LIBRARIES Imath::Imath)
 else()
     macro_optional_find_package(OpenEXR)
 endif()
-- 
GitLab

